src/bootstrap/src/core/builder.rs

```diff
@@ -1687,6 +1687,16 @@ impl<'a> Builder<'a> {
             }
         }
 
+        if self.config.rust_strip {
+            rustflags.arg("-Cstrip=symbols");
```

Comment:
If possible (and it should be here) we prefer to let Cargo manage the passing of arguments. Can we set an environment variable for Cargo instead here? https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#strip

That usually plays better with appropriately threading it through.

Reply:
Did you mean I could use  `cargo.env(profile_var("STRIP"), &self.config.rust_strip);` ?

Reply:
> If possible (and it should be here) we prefer to let Cargo manage the passing of arguments. Can we set an environment variable for Cargo instead here? https://doc.rust-lang.org/nightly/cargo/reference/profiles.html#strip
> 
> That usually plays better with appropriately threading it through.

Did you mean I could use cargo.env(profile_var("STRIP"), &self.config.rust_strip); ?

Reply:
Did you mean I could use `cargo.env(profile_var("STRIP"), &self.config.rust_strip); ` ?